src/authentication/authentication.graphql

```diff
@@ -1,6 +1,10 @@
 type Mutation {
     passwordLogin(input: UserPasswordLoginInput!): TokenResponse
     passwordSignup(input: UserPasswordSignupInput!): UserSignupResponse
+    inviteTokenSignup(input: UserInviteTokenSignupInput): InviteTokenResponse
+    refreshInviteToken(id: ID!):InviteTokenResponse
+    setPasswordForInvite(input: UserPasswordForInviteInput): UserSignupResponse
+    revokeInviteToken(id: ID!): String
```

Comment:
Let's ensure it returns a mandatory value and not null in graphql. Same with the other mutations as well.